### ISSUE TITLE
Feature request: Support HTTP namespaces for service discovery

### ISSUE THREAD

**Issue by WKuipers** [terraform-providers/terraform-provider-aws#6783]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Please add support for HTTP-only namespaces in AWS service discovery as specified by https://docs.aws.amazon.com/cloud-map/latest/dg/namespaces-values.html in "API Calls" or the creation call documented at https://docs.aws.amazon.com/cloud-map/latest/api/API_CreateHttpNamespace.html

### New or Affected Resource(s)

aws_service_discovery

### Potential Terraform Configuration

```hcl
resource "aws_service_discovery_http_namespace" "example" {
  name        = "somename"
  description = "example"
}
```

**Comment by saravanan30erd** [comment]
I will work on this.

**Comment by bflad** [comment]
Support for this new resource has been merged in and will release with version 1.53.0 of the AWS provider, likely tomorrow. 👍
